crates/bevy_render/src/color/mod.rs

```diff
@@ -47,11 +47,15 @@ pub enum Color {
     },
 }
 
+// List of HTML colors sourced from https://www.w3schools.com/tags/ref_colornames.asp.
+// Extended color keywords license: https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document.
```

Comment:
We need to actually follow this license, not just link it. Also, I'm afraid I linked the wrong license - it's actually this one https://www.w3.org/Consortium/Legal/2015/doc-license (linked from https://www.w3.org/TR/css-color-4/#named-colors:
![image](https://user-images.githubusercontent.com/36049421/172949340-af08d3aa-e491-48d6-9455-8bcdf9001297.png)
)
I *think* we need:
> [Copyright](https://www.w3.org/Consortium/Legal/ipr-notice#Copyright) © 2022 [W3C](https://www.w3.org/)® ([MIT](https://www.csail.mit.edu/), [ERCIM](https://www.ercim.eu/), [Keio](https://www.keio.ac.jp/), [Beihang](https://ev.buaa.edu.cn/)). W3C [liability](https://www.w3.org/Consortium/Legal/ipr-notice#Legal_Disclaimer), [trademark](https://www.w3.org/Consortium/Legal/ipr-notice#W3C_Trademarks) and [permissive document license](https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document) rules apply.

However, this also adds new restrictions to the final game created (so long as any of these colour constants get compiled in?), so it might be better to move it to a seperate crate entirely.

Reply:
This seperate crate would need this license item https://spdx.org/licenses/W3C-20150513.html